backend/python/app/middlewares/auth.py

```diff
@@ -222,6 +225,18 @@ def wrapper(root, info, *args, **kwargs):
                             story_translation_content_id=story_translation_content_id,
                         )
                     )
+            elif "id" in kwargs:
```

Comment:
hmmmmmm. is this change for only the SoftDeleteStoryTranslation mutation? i dont like that `id` is so ambiguous, would you mind changing SoftDeleteStoryTranslation's param `id` to `story_translation_id`? then I dont think youd need this new elif case, and there's less room for error since it's a clear parameter name.

that being said youll probably have to hunt for where the the mutation is used in the front end and rename the param passed in

Reply:
That sounds like a good idea, I felt pretty iffy about my solution.